packages/discussions/src/types.ts

```diff
@@ -767,13 +767,14 @@ export interface IChannelAclPermission
  * @interface ICreateChannelSettings
  */
 export interface ICreateChannelSettings {
+  allowedReactions?: PostReaction[];
+  allowReaction?: boolean;
   allowReply?: boolean;
-  softDelete?: boolean;
+  blockWords?: string[];
   defaultPostStatus?: PostStatus;
-  allowReaction?: boolean;
+  guidelineUrl?: string;
```

Comment:
See comments from @rgwozdz and myself on [Discussions API PR-394](https://devtopia.esri.com/dc/hub-discussions/pull/394) concerning `metadata` (or similar) vs solitary `guidelineUrl` property.

Reply:
👍